flix-core/src/main/resources/db.migration/V__1.1__create_user_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE USER (
+  id        INTEGER,
+  login     VARCHAR(1000),
+  password  VARCHAR(50),
+  age       INTEGER,
+  gender    VARCHAR(10)
```

Comment:
PRIMARY KEY (USER_ID) - this should happen automatically, but better safe than sorry :)

Reply:
Done